test/test_crypten.py

```diff
@@ -283,12 +283,8 @@ def forward(self, input):
         return out
 
     def set_all_parameters(self, value):
-        self.fc1.weight.data.fill_(value)
-        self.fc1.bias.data.fill_(value)
-        self.fc2.weight.data.fill_(value)
-        self.fc2.bias.data.fill_(value)
-        self.fc3.weight.data.fill_(value)
-        self.fc3.bias.data.fill_(value)
+        for p in self.parameters():
```

Comment:
Q: Is there any reason why the attribute ```data.fill_``` is used?

Reply:
As far as I'm aware, `torch.nn.init.constant_(param, value)` is the same as `param.data.fill_(value)`.

constant: https://pytorch.org/docs/stable/nn.init.html#torch.nn.init.constant_
fill: https://pytorch.org/docs/stable/tensors.html#torch.Tensor.fill_

Reply:
They should do the same thing but changed it to ```constant_``` because of an issue indicated in this [PR](https://github.com/facebookresearch/CrypTen/pull/112)